### DISCUSSION TITLE
Tdarr Node

### DISCUSSION THREAD

**Discussion by Rocketplanner83** [tteck/Proxmox#3477] [Ideas]
Can we get an LXC for Tdarr as a NODE to connect to another server?

**Comment by Narkowii** [comment]
yes

**Reply by Rocketplanner83** [reply]
Wonderful!

**Reply by Narkowii** [reply]
For all LXC node only, i disable the server => i just remove the file tdarr-server.service in /etc/systemd/system/

**Reply by Rocketplanner83** [reply]
Okay, so how do you configure the file for the node?

**Reply by Narkowii** [reply]
Open the file Tdarr_Node_Config.json and put your Tdarr IP server

**Comment by Rocketplanner83** [comment]
Ive been struggling to get this Node to work with my Tdarr server running on my TrueNas Box.  

I set up NFS shares on truenas and configured the fstab in the lxc but I still am having an issue with the lxc being able to connect to trueNAS.  I am kind of perplexed.
